src/management/projects.ts

```diff
@@ -0,0 +1,16 @@
+export interface FirebaseProjectMetadata {
+  name: string /* The fully qualified resource name of the Firebase project */;
+  projectId: string;
+  projectNumber: string;
+  displayName: string;
+  resources: DefaultResource;
+}
+
+export interface DefaultResource {
```

Comment:
"DefaultResource" seems... very broad. How about `DefaultProjectResources`? (So when it's required in other files, it's referenced as "DefaultProjectResources")

Reply:
Yeah, SGTM!